app/queries/servers/categories.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Database, Q} from '@nozbe/watermelondb';
+
+import {MM_TABLES} from '@constants/database';
+
+import type ServerDataOperator from '@database/operator/server_data_operator';
+import type CategoryModel from '@typings/database/models/servers/category';
+
```

Comment:
There are numerous queries in this file.  Are all of them in use?  If not, as a suggestion, can we keep only the queries that we need so as not to have 'redundant' functions.  ( being minimalist ) ?